lib/friendly_shipping/services/ups_json/generate_city_state_lookup_payload.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module FriendlyShipping
+  module Services
+    class UpsJson
+      class GenerateCityStateLookupPayload
+        def self.call(location:)
+          {
+            XAVRequest: {
+              RegionalRequestIndicator: "1",
```

Comment:
This is documented as taking a true or false (https://developer.ups.com/api/reference?loc=en_US#tag/Address-Validation). Does it work without this? Or is this covered by the reequestoption of 1 in the path `/v2/1`?

I would trust the actual behavior over the docs though!

Reply:
This is actually the indicator that lets us get away with not including a full address in the request. I had to email them to ask how we could pass a zip by itself to the API and get back a city and state, and they said we had to pass this `RegionalRequestIndicator` option with a value of "1" along with a zip code.

Don't ask me why it's this way! Makes no sense. But without this option, the API complains about us providing an incomplete address when we just pass the zip code and no city/state/address.